### PR TITLE
Jonas modify clear all command

### LINKED ISSUE #44: As a new user, I want to clear all example sample address,
so that I can start using the application quickly.

### PR DESCRIPTION
**Pull request by Jonaspng**

### CODE REVIEW

**Review by normkoh — COMMENTED** [review]
LGTM! Although it seems like users can bypass the confirmation by using "clear-confirm" straightaway.

**Comment by nitant-p** [comment]
LGTM!

File: src/main/java/seedu/address/logic/commands/ClearCommand.java [hunk 1]
```diff
@@ -10,13 +11,18 @@
  */
 public class ClearCommand extends Command {
 
-    public static final String COMMAND_WORD = "clear";
+    public static final String COMMAND_WORD = "clear-confirm";
     public static final String MESSAGE_SUCCESS = "Address book has been cleared!";
-
+    public static final String MESSAGE_CONFIRMATION = "You are attempting to clear all addresses in the memory."
+            + " Add '-confirm' to clear all addresses.";
```
Comment (GarethOng) [inline src/main/java/seedu/address/logic/commands/ClearCommand.java]: How about changing it to 
```Add '-confirm' to the end of the command```
 to make the instruction less ambiguous?

Comment (Jonaspng) [reply src/main/java/seedu/address/logic/commands/ClearCommand.java]: Hmm good point. I will edit it. Thanks.

**Review by JJoeYi — COMMENTED** [review]
LGTM

**Pull request merged**